pineapple-bundle/pineapple-core/src/main/java/sh/miles/pineapple/config/type/FieldModifier.java

```diff
@@ -0,0 +1,24 @@
+package sh.miles.pineapple.config.type;
+
+import java.lang.reflect.Field;
+
+public class FieldModifier {
+
+    public static final FieldModifier INSTANCE = new FieldModifier();
+
+    public Object getValue(Field field, Object instance) {
```

Comment:
These methods can be static unless their is a reason they can't be? It doesn't look like you went with the approach I suggested, so wrapping this in a singleton seems pointless.